networkx/classes/ordered.py

```diff
@@ -0,0 +1,55 @@
+"""
+OrderedDict variants of the default base classes.
+
+These classes are especially useful for doctests and unit tests.
```

Comment:
"Using these classes is an especially bad idea for doctests and unit tests" :smirk: